verticapy/tests_new/core/parsers/test_pandas_extended.py

```diff
@@ -0,0 +1,78 @@
+"""
+Copyright  (c)  2018-2024 Open Text  or  one  of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+
+# Pytest
+import pytest
+
+# Standard Python Modules
+import time
+
+from vertica_python.errors import CopyRejected
+
+# Other Modules
+import pandas
+
+# VerticaPy
+from verticapy import (
+    drop,
+)
+from verticapy.connection import current_cursor
+from verticapy.core.parsers.pandas import read_pandas
+from verticapy.datasets import load_titanic
+
+
+@pytest.fixture(scope="module")
```

Comment:
We have created a conftest file under ~/VerticaPy/verticapy/tests_new/conftest.py which has variety sets of data. Can we use that instead of creating a new one?